lib/utils.js

```diff
@@ -131,6 +131,9 @@ const getOsInfo = _.memoize(
   }
 );
 
+// @ts-expect-error
+// error TS2345: Argument of type '{}' is not assignable to parameter of type 'DriverOpts<Readonly<Record<string, Constraint>>>'
+// Type '{}' is missing the following properties from type 'ServerArgs': address, allowCors, allowInsecure, basePath, and 26 more.
```

Comment:
This PR just ignores the tsc error as a workaround but probably I think it'd be better to expose [generateDriverLogPrefix](https://github.com/appium/appium/blob/943dce5eec4c047073ef9dfcb97da9ccd6be9459/packages/base-driver/lib/basedriver/helpers.js#L596) function.